### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#137] [General]
# index

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/

**Comment by onenewcode** [comment]
支持大佬，以star

**Comment by abelce** [comment]
支持一下

**Comment by onenewcode** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by sanbei101** [comment]
支持

**Comment by hexody** [comment]
已经出版了吗？

**Reply by isno** [reply]
没有啊，一审没通过，有不少错别字，文笔严谨性不行，还得细改。

**Comment by xiongyunhua-star** [comment]
支持支持， 为作者点赞

**Comment by 1209233066** [comment]
来找错别字了

**Comment by bitwool** [comment]
纸质书和在线版主要区别在什么地方呢

**Reply by isno** [reply]
纸质版经过了编辑的校验还有排版，更适合阅读。

**Comment by cxok** [comment]
图 1-6 XaaS 演进  ，这个图中 ，saas部分画得有问题
